Project 2/four-card-feature/style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 1440px;
+  font-size: 15px;
+  font-family: "Poppins", sans-serif;
+  color: hsl(234, 12%, 34%);
+  background: hsl(0, 0%, 98%);
+  margin: 20px auto;
+}
+
+.smaller-text {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+}
+.header {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+}
+
+.Boxcontainer {
+  display: flex;
+  align-items: center;
+  padding: 20px;
+  justify-content: center;
+  flex-wrap: wrap;
+  overflow-wrap: break-word;
```

Comment:
to mogłoby zadziałać dla tekstu, ale nadajesz to na kontener z boxami, więc nie ma żadnego efektu :)